x11-terms/terminator/terminator-1.92.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1 virtualx xdg-utils
+
+DESCRIPTION="Multiple GNOME terminals in one window"
+HOMEPAGE="https://github.com/gnome-terminator/terminator"
+SRC_URI="https://github.com/gnome-terminator/terminator/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="dbus +libnotify"
+
+RDEPEND="
+	>=dev-libs/glib-2.32:2
+	dev-libs/keybinder:3[introspection]
+	dev-python/configobj[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	>=x11-libs/gtk+-3.16:3
+	x11-libs/vte:2.91[introspection]
+	dbus? ( sys-apps/dbus )
+	libnotify? ( x11-libs/libnotify[introspection] )
+"
+BDEPEND="
+	dev-util/intltool
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.91-without-icon-cache.patch
+	"${FILESDIR}"/${PN}-1.91-desktop.patch
+)
+
+src_prepare() {
+	xdg_environment_reset
+	distutils-r1_src_prepare
+}
+
+python_test() {
+	virtx esetup.py test
```

Comment:
Please put `|| die "tests fail with ${EPYTHON}"` here

Reply:
Done. Though I think it should be done in the eclass instead.

Reply:
Actually, it is done in the eclass.

Also, it's better to do:

```
src_test() {
    virtx distutils-r1_src_test
}
```

as that avoids starting X n times. It would also save you from redefining `python_test`.

Reply:
You didn't read the bible: https://dev.gentoo.org/~mgorny/python-guide/distutils.html#customizing-the-test-phase

Reply:
Ah, thanks. I didn't know such bible existed. Will read.

Sent you #15589